m4/ax_boost_base.m4

```diff
@@ -161,7 +161,7 @@ if test "x$want_boost" = "xyes"; then
         AC_MSG_RESULT(yes)
     succeeded=yes
     found_system=yes
-        ],[
+        ],[:
```

Comment:
If this is an improvement to ax_booost_base.m4 maybe it should be contributed back to https://github.com/autoconf-archive/autoconf-archive

Reply:
As an alternative suggestion to @jamesba's; Is this related to #25? If so, would be be appropriate to remove this patch and fix the line endings?